lib/src/commons/BrowserUtils.js

```diff
@@ -277,8 +277,8 @@ var BrowserUtils;
         chillOut();
         isExist(selector);
         Reporter_1.Reporter.debug(`Switching to an Iframe by selector '${selector}'`);
-        tryBlock(() => browser.switchToFrame($(selector)), 'Failed to switch frame');
         chillOut();
```

Comment:
check if we still need it with v5

Reply:
removed the one before isExist left the one after switch

Reply:
but do we still need it?